src/config/schema.ts

```diff
@@ -428,7 +428,7 @@ export const configSchema = {
       additionalProperties: false,
       default: {
         initialStatus: "active",
-        initialWallets: ["BTC"],
+        initialWallets: ["BTC", "USD"],
```

Comment:
@nicolasburtey 

Should we add USD wallet as default? I think its default in our deployments config), also it will help with mobile app device account testing to avoid the `2 wallets error` @UncleSamtoshi